data/json/itemgroups/ammo.json

```diff
@@ -237,6 +237,78 @@
     ]
   },
   {
+    "type": "item_group",
+    "id": "vending_ammo_pistol",
+    "items": [
+      [ "22_fmj", 9 ],
+      [ "22_lr", 27 ],
+      [ "22_cb", 24 ],
```

Comment:
I'm not sure I like the requirement to provide a duplicate definition for some (majority) ammo types. You might best having vending machines dispense `ammo_common`. Having more disparate groups isn't of much benefit for a much higher maintenance cost.